Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM meedan/nodejs
+FROM node:10
```

Comment:
Why 10? Is it the version we are running in Lambda? Did you try with Node 12?

Reply:
It's what's running in Lambda. I'm happy to change to 12.

Reply:
Makes sense to keep in sync the Dockerfile and what we are running on Lambda. When we upgrade, we upgrade both.